libraries/rush-lib/src/api/SubspaceConfiguration.ts

```diff
@@ -4,14 +4,15 @@
 import { FileSystem, JsonFile, JsonSchema } from '@rushstack/node-core-library';
 
 import type { RushConfiguration } from './RushConfiguration';
+import type { RushConfigurationProject } from './RushConfigurationProject';
 import schemaJson from '../schemas/subspaces.schema.json';
 import { RushConstants } from '../logic/RushConstants';
 
 /**
  * The allowed naming convention for subspace names.
  * Allows for names to be formed of letters, numbers, and hyphens (-)
  */
-export const SUBSPACE_NAME_REGEXP: RegExp = new RegExp('/^[a-z][a-z0-9]*([-][a-z0-9]+)*$/');
+export const SUBSPACE_NAME_REGEXP: RegExp = new RegExp(/^[a-z][a-z0-9]*([-][a-z0-9]+)*$/);
```

Comment:
Isn't `new RegExp(/xyz/)` equivalent to `/xyz`?

Reply:
Changed